### ISSUE TITLE
Feature Request Album cover visualization

### ISSUE THREAD

**Issue by gfcapalbo** [clementine-player/Clementine#5230]
Fantastic feature request.

The °show fullsize° feature for covers is OK but shows a 600x600 window.
Most people like to use all screen estate to look at the cover when listening to music and not multitasking-

A visualization option that just shows the cover , stretched over full screen would be really really cool.

**Comment by santigl** [comment]
Right now that is implemented using a `QDialog` that is created, displayed and just waits until you close it. It doesn't have the capabilities of updating the image or responding to other events.

In order to implement that, we would need to create a new class. Since we would have a dedicated handler for that window, it would probably end up making sense to also allow to show lyrics, and maybe also display playback controls.

Is that something that could be added?

**Comment by narunlifescience** [comment]
this is not that hard to implement.. but stretching an image to full screen will distort it completely with respect to aspect ratio & resolution.. so i don't think this is a good idea.

**Comment by santigl** [comment]
That's true. If it has a relatively big size it could be displayed 1:1 and centered. The problem would be with small covers...

**Comment by gfcapalbo** [comment]
i was thinking about a non-deforming zoom.  Images bigger than the monitor resolution would be  reduced, also without deforming.
If there is unbearable quality loss, the user could just use the current "show fullsize" , or perhaps download a higher res cover.

The whole reason for this request is emulating the old pleasure of enjoying the large LP pictures while listnening to music.

**Comment by alexanderdd** [comment]
Hey, there is a related issue here https://github.com/clementine-player/Clementine/issues/1284
